### PR TITLE
Move integrations to packs

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] In Progress

## Description
Moving integrations from Legacy pack to pack per integration

For the pack metadata, filled in the following fields from the yaml:
 - name - integration display name
 - fromversion
 - category
 - tags - integration display name
 - deprecated

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

File: Packs/AWS_DynamoDB/pack_metadata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "support": "Cortex XSOAR",
+    "serverMinVersion": "1.0.0",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": "IT Services",
```
Comment (glicht) [inline Packs/AWS_DynamoDB/pack_metadata.json:10]: I think this should be an array

File: Packs/AWS_DynamoDB/pack_metadata.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "tags": "Amazon DynamoDB",
```
Comment (glicht) [inline Packs/AWS_DynamoDB/pack_metadata.json:11]: I think this should be an array

**Comment by yuvalbenshalom** [comment]
where is the issue for this?
why is this in one PR?

**Comment by Itay4** [comment]
closing this, will be done gradually

**Pull request closed without merging**